src/funnies.js

```diff
@@ -234,5 +234,8 @@ export default [
   "Discovering new ways of making you wait...",
   "Sorry we are busy catching em' all, we're done soon",
   "TODO: Insert elevator music",
-  "Still faster than Windows update"
+  "Still faster than Windows update",
+  "Is your body from McDonalds? Because i'm loving it",
```

Comment:
`Is your body from McDonalds? Because i'm loving it`

Eh, I feel like that could be interpreted the wrong way. Not a big fan of this one.